components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -811,12 +811,17 @@ define([
                     this.getAgeLabel().show();
                     labels.push(this.getAgeLabel());
                 }
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
+                if (this.getCommentsLabel()) {
+                    this.getCommentsLabel().show();
+                    labels.push(this.getCommentsLabel());
+                }
             } else {
                 this.getNameLabel() && this.getNameLabel().hide();
                 this.getAgeLabel() && this.getAgeLabel().hide();
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
```

Comment:
AH, I see, I did not realize what happened. You are right, order is important in this case. But I would probably stop the if-else, do one line for externalID, do another if-else specifically for comments. Code duplication is IMHO worse and less intuitive and less maintainable than an extra if